ModelBundle/Repository/RouteDocumentRepository.php

```diff
@@ -23,6 +23,16 @@ public function findOneByName($name)
     }
 
     /**
+     * @param string $redirectionId
+     *
+     * @return RouteDocumentInterface
+     */
+    public function findByRedirection($redirectionId)
+    {
+        return $this->findBy(array('name' => new \MongoRegex('/.*_' . $redirectionId . '/')));
```

Comment:
If two ids are presents in the base: ``somevalue`` and ``someprefix_somevalue`` and we try to ``findByRedirection('somevalue')``, which result is served ?